src/Stats.h

```diff
@@ -6,6 +6,23 @@
 namespace mms {
 
 enum class StatsEnum{
+    SPEED_RUNLEFT,
+    SPEED_RUNRIGHT,
+    SPEED_TURNLEFT45,
+    SPEED_TURNRIGHT45,
+    SPEED_HALFFORWARD,
+    SPEED_EDGEFORWARD,
+    SPEEDSUM,
```

Comment:
I don't know what any of these mean. Can you include a comment above each one describing what it's keeping track of?